lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.call(fire)
```

Comment:
`return deathStar.fire.call(alderaan);`

deathstar has the function fire. `.call` sets the value of `this` to the scope of alderaan